packages/core/core/src/PackagerRunner.js

```diff
@@ -665,7 +665,8 @@ export default class PackagerRunner {
         bundleGraph.getHash(bundle) +
         JSON.stringify(configResults) +
         JSON.stringify(globalInfoResults) +
-        this.options.mode,
+        this.options.mode +
+        `${this.options.shouldBuildLazily ? 'lazy' : 'eager'}`,
```

Comment:
/nit but this doesn't really need to be a template string.. (but I get that you probably just copied what you did in all the other places, no big deal..)